### PR TITLE
oscavmgr: init at 0.4.1

### PR DESCRIPTION
**Pull request by Scrumplex**
## Description of changes

Utility for face tracking for VRChat and Resonite. Not very useful for VRChat unless [VrcAdvert](https://github.com/galister/VrcAdvert) is also packaged.

https://github.com/galister/oscavmgr

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/os/oscavmgr/package.nix
Comment (Pandapip1) [inline pkgs/by-name/os/oscavmgr/package.nix:55]:
```suggestion
    changelog = "https://github.com/galister/oscavmgr/releases";
```
I strongly prefer this, but I won't block on it as there is a lack of consensus

File: pkgs/by-name/os/oscavmgr/package.nix
Comment (Pandapip1) [inline pkgs/by-name/os/oscavmgr/package.nix:58]:
```suggestion
    platforms = lib.platforms.unix;
```
Can I confirm that this is linux-only and not generally unix-compatible (e.g. the BSDs)?

File: pkgs/by-name/os/oscavmgr/package.nix
Comment (Pandapip1) [inline pkgs/by-name/os/oscavmgr/package.nix:57]: Feel free to add me as a maintainer if you want.

**Review by Pandapip1 — APPROVED** [review]
Diff LGTM.

File: pkgs/by-name/os/oscavmgr/package.nix
Comment (Scrumplex) [reply pkgs/by-name/os/oscavmgr/package.nix:58]: It can probably build on other platforms, but the applications it's designed to work with only run on Windows and Linux realistically.

**Review by Pandapip1 — APPROVED** [review]
Ofborg re-requested my review

**Comment by Pandapip1** [comment]
FYI - VRCAdvert is also packaged.

**Pull request merged**
